docs/api.asciidoc

```diff
@@ -249,7 +249,7 @@ the gem and using the `Railtie`.
 
 [source,ruby]
 ----
-ElasticAPM::Rails.start(server_url: 'http://localhost:8200')
+ElasticAPM::Rails.start(MyRailsApp.new, server_url: 'http://localhost:8200')
```

Comment:
Aren't there a bunch of side effects from calling `new` on an app?

Reply:
I'd be a little bit concerned about `new`ing a rails app. Too much magic going on 😅

Reply:
If a user doesn't use a Railtie, then how do they initialize a Rails app? A user would pass their app instance to the `ElasticAPM::Rails.start` method and however they instantiate it is up to them. Do you know of a better example of instantiating a rails app without a Railtie?

Reply:
I might be misunderstanding, but I don't think it's our job to start nor instantiate users' apps?

If someone wants to use this API, I think they should do so in an initializer or their own custom railtie. And when they do, they'll have the application object at hand. I don't think we ought to encourage any other way to start the agent with Rails.

Reply:
Yes, we are saying the same thing. I was just trying to provide a simple example. So would this be better? The `myapp` object is whatever application object they have in an initializer or custom railtie.

`ElasticAPM::Rails.start(myapp, server_url: 'http://localhost:8200')`

Reply:
I think so. Or maybe include a whole example of starting in an initializer (including a note that this is the hard way)

Reply:
ok I added an example initializer